cpp/arcticdb/storage/storage_factory.cpp

```diff
@@ -54,6 +57,14 @@ std::shared_ptr<VariantStorageFactory> create_storage_factory(
         res = std::make_shared<VariantStorageFactory>(
             nfs_backed::NfsBackedStorageFactory(nfs_backed_config)
         );
+#ifndef ARCTICDB_USING_CONDA
+    } else if (type_name == azure::AzureStorage::Config::descriptor()->full_name()) {
+        azure::AzureStorage::Config azure_config;
+        storage.config().UnpackTo(&azure_config);
+        res = std::make_shared<VariantStorageFactory>(
+            azure::AzureStorageFactory(azure_config)
+        );
+#endif
```

Comment:
It will be very helpful to add an `#else` here to explain Azure is not supported and a link to the GitHub issue for the user to follow.

Reply:
Added